pkg/oam/util/helper.go

```diff
@@ -297,6 +299,23 @@ func GetDefinitionName(dm discoverymapper.DiscoveryMapper, u *unstructured.Unstr
 	return mapping.Resource.Resource + "." + groupVersion.Group, nil
 }
 
+// GetGVKFromDefinition help get Group Version Kind from DefinitionReference
+func GetGVKFromDefinition(dm discoverymapper.DiscoveryMapper, definitionRef v1alpha2.DefinitionReference) (schema.GroupVersionKind, error) {
```

Comment:
I don't see any place that uses this method.

Reply:
Could you provide an example how this is gonna be used?

Reply:
sorry, I added one in the code.